tests/test_model_serializer.py

```diff
@@ -1301,6 +1301,11 @@ class Issue6751Model(models.Model):
     char_field2 = models.CharField(max_length=100)
 
 
+class Issue2648Model(models.Model):
+    char_field = models.CharField(max_length=100)
+    updated_at = models.DateTimeField(auto_now=True, blank=True, null=True)
+
```

Comment:
I belive that it would be better to test a model with at least 3 fields because with only 2 fields there is no difference between giving the extra-update-fields and not.